pyproject.toml

```diff
@@ -1,3 +1,40 @@
+[project]
+name = "natcap.invest"
+description = "InVEST Ecosystem Service models"
+readme = "README_PYTHON.rst"
+requires-python = ">=3.8,<3.11"
+license = {file = "LICENSE.txt"}
+maintainers = [
+    {name = "James Douglass", email = "jdouglass@stanford.edu"}
+]
```

Comment:
I just copied this over from `setup.py`, but totally understand if you're not comfortable with it!
One benefit of PEP 621/pyproject.toml is that `maintainers` is an array, so we can include any number of people. (in setup.py, `maintainer` is just one string). From the PEP:
> The people or organizations considered to be the “authors” of the project. The exact meaning is open to interpretation — it may list the original or primary authors, current maintainers, or owners of the package.
The “maintainers” field is similar to “authors” in that its exact meaning is open to interpretation.

So we could certainly include multiple people, or just a single contact for the organization.

Reply:
>I just copied this over from setup.py, but totally understand if you're not comfortable with it!
Oh yeah, not to worry!  I had just forgotten that this was a conversation I wanted to have, and I was just reminded about it because of this move, that's all.

I'm glad to hear that this is now an array!  Much more realistic, I think.

Reply:
For what it's worth, we do have a software team list on lists.stanford.edu, I think it's `natcap-software@lists.stanford.edu`.  I created it at UIT's suggestion back when we got our first code-signing cert, but we haven't used it since.  Could be useful _if_ we wanted emails to the group to go to the whole group.

Reply:
Oh cool! Now that I'm thinking about it, is an email required? Is providing an email to an open source project something that other projects do?

Reply:
I'll have to defer on whether the email is required here, but providing email addresses is common in open source and definitely in the python community

Reply:
It seems like the `natcap-software` list would be a good choice if we want to provide an email. And I agree the name could be "Natural Capital Project". We're all already listed as maintainers on PyPI, so there doesn't seem much added benefit to listing all our names in this file? https://pypi.org/project/natcap.invest/

Reply:
That sounds good, I don't have any strong preference.